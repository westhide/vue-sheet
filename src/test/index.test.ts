export default describe("suite name", () => {
  it("init", () => {
    expect(1 + 1).toEqual(2);
    console.log("vitest init");
  });

  it("snapshot", () => {
    expect(1 + 1).toMatchSnapshot();
  });
});
