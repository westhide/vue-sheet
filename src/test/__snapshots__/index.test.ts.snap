// Vitest Snapshot v1

exports[`suite name > snapshot 1`] = `2`;
